src/uct/ib/base/ib_md.c

```diff
@@ -1402,6 +1411,20 @@ uct_ib_md_parse_subnet_prefix(const char *subnet_prefix_str,
     return UCS_OK;
 }
 
+static double uct_ib_md_pci_bw(const uct_ib_md_config_t *md_config,
+                               struct ibv_device *ib_device)
+{
+    unsigned i;
+
+    for (i = 0; i < md_config->pci_bw.count; i++) {
+        if (!strcmp(ib_device->name, md_config->pci_bw.device[i].name)) {
+            return md_config->pci_bw.device[i].bw * 0.9846; /* ~1.54% lost for PCI Gen3 */
```

Comment:
pls add formula which was used to calculate 0.9846 according to gen3 spec (e.g 64/66*128/148... etc)

Reply:
Info from this article: https://www.tested.com/tech/457440-theoretical-vs-actual-bandwidth-pci-express-and-thunderbolt/

Reply:
added explanation